lib/issues2eslint.js

```diff
@@ -252,12 +256,22 @@ class MythXIssues {
 
         if (sourceFormat === 'evm-byzantium-bytecode') {
             // Pick out first byteCode offset value
-            const offset = parseInt(issue.sourceMap.split(':')[0], 10);
-            [startLineCol, endLineCol] = this.byteOffset2lineColumn(offset, lineBreakPositions);
+            if (issue.sourceMap) {
+                const offset = parseInt(issue.sourceMap.split(':')[0], 10);
+                [startLineCol, endLineCol] = this.byteOffset2lineColumn(offset, lineBreakPositions);
+            } else {
+                startLineCol = { line: -1, column: 0 };
+                endLineCol = { line: -1, column: 0 };
+            }
         } else if (sourceFormat === 'text') {
             // Pick out first srcEntry value
-            const srcEntry = issue.sourceMap.split(';')[0];
-            [startLineCol, endLineCol] = this.textSrcEntry2lineColumn(srcEntry, lineBreakPositions);
+            if (issue.sourceMap) {
+                const srcEntry = issue.sourceMap.split(';')[0];
+                [startLineCol, endLineCol] = this.textSrcEntry2lineColumn(srcEntry, lineBreakPositions);
+            } else {
+                startLineCol = { line: -1, column: 0 };
+                endLineCol = { line: -1, column: 0 };
```

Comment:
We can make this section a bit more concise by swapping the if statements around some:
```
if(issue.sourceMap)
    if sourceFormat == evm-byzantium-bytecode
        // set startLineCol, endLineCol
    else if sourceFormat == text
        // set startLineCol, endLineCol
else
    startLineCol = { line: -1, column: 0 };
    endLineCol = { line: -1, column: 0 }; 
```

Reply:
Done!